ui/src/app/pages/fleet/agents/list/agent.list.component.ts

```diff
@@ -120,25 +120,33 @@ export class AgentListComponent implements OnInit, AfterViewInit, AfterViewCheck
     this.columns = [
```

Comment:
hm I know this would require a bigger change, but I like using this with columns and tables with CSS
it helps specify each data goes where. For future notes
https://www.w3schools.com/css/css_grid_container.asp

Reply:
:+1: thx, the listing def. needs an overhaul. lets discuss it in more detail.